### DISCUSSION TITLE
Error as i-simpa starts

### DISCUSSION THREAD

**Discussion by gasgiovinco75** [Ifsttar/I-Simpa#359] [General]
Dear All,
when I open the software (ver 1.3.4 under Windows 11), I have the following message:

Error:Unable to init the Python(TM) element MD_Octave of the md_octave module

Have any of you had the same error?
Have you solved it?

Thanks a lot in advance.

**Comment by nicolas-f** [comment]
Hi,

This is with old project that contains reference to non available computation core. This is only a warning and your project is just working fine.

regards,
